src/main/java/com/studioedge/focus_to_levelup_server/domain/focus/service/FocusService.java

```diff
@@ -144,8 +144,13 @@ public void saveFocus(Member m, Long subjectId, SaveFocusRequest request) {
             gm.addWeeklyFocusTime(request.focusSeconds());
         }
 
+        // 하루 최대 집중시간 확인하기
+        if (dailyGoal.getMaxConsecutiveSeconds() > request.maxConsecutiveSeconds()) {
```

Comment:
```suggestion
        if (dailyGoal.getMaxConsecutiveSeconds() < request.maxConsecutiveSeconds()) {
```
이거 조건이 반대로 되어있는 것 같습니다.